locale/sw.yml

```diff
@@ -0,0 +1,1813 @@
+  divmod: "divmod"
+  enumerate: "kuhesabu"
+  eval: "tathmini"
+  exec: "tekeleza"
+  filter: "chuja"
+  float: "elea"
+  format: "muundo"
+  frozenset: "waliohifadhiwa"
+  getattr: "pata-sifa"
+  globals: "ulimwengu"
+  hasattr: "ina-sifa"
+  hash: "hash"
+  help: "saidia"
+  hex: "hex"
+  id: "kitambulisho"
+  input: "pembejeo"
+  int: "nambari"
+  isinstance: "mfano"
+  issubclass: "darasa-ndogo"
+  iter: "mara-kwa-mara"
+  len: "urefu"
+  list: "orodha"
+  locals: "wenyeji"
+  map: "ramani"
+  max: "upeo"
+  memoryview: "mtazamo-wa-kumbukumbu"
+  min: "kiwango-cha-chini"
+  next: "ijayo"
+  object: "kitu"
+  oct: "oct"
+  open: "fungua"
+  ord: "utaratibu"
+  pow: "nguvu"
+  print: "chapisha"
+  property: "mali"
+  range: "masafa"
+  repr: "repr"
+  reversed: "geuza-nyuma"
+  round: "raundi"
+  set: "seti"
+  setattr: "weka-sifa"
+  slice: "kipande"
+  sorted: "pangwa"
+  staticmethod: "njia-tuli"
+  str: "str"
+  sum: "jumla"
+  super: "super"
+  tuple: "tuple"
+  type: "aina"
+  vars: "vars"
+  zip: "bana"
+  __import__: "_agiza_"
+
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE: RAZOR   |
+# ----------------------
+# << Keywords >>
+page: "" # (Requires ASP.NET Core 2.0 and later)
+namespace: "" #
+functions: "" #
+inherits: "" #
+model: "" #
+section: "" #
+helper: "" # (Not currently supported by ASP.NET Core)
+@case: "" #
+@do: "" #
+@default: "" #
+@for: "" #
+@foreach: "" #
+@if: "" #
+@else: "" #
+@lock: "" #
+@switch: "" #
+@try: "" #
+@catch: "" #
+@finally: "" #
+@using: "" #
+@while: "" #
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE: REGEX  |
+# ----------------------
+# << Keywords >>
+# Not sure how to template that out // taking all ideas
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE: RUBY   |
+# ----------------------
+# << Keywords >>
+alias: "jina-mbadala"
+and: "na"
+begin: "anza"
+break: "vunja"
+case: "kisa"
+class: "darasa"
+def: "fanya"
+defined?: "ilidefiniwa?"
+do: "fanya"
+else: "vinginevyo"
+elsif: "au-kama"
+end: "maliza"
+ensure: "hakikisha"
+false: "uwongo"
+for: "kwa"
+if: "kama"
+in: "katika"
+module: "moduli"
+next: "fuata"
+nil: "siyo"
+not: "si"
+or: "au"
+redo: "rudia"
+rescue: "okoa"
+retry: "jaribu-tena"
+return: "rejesha"
+self: "mwenyewe"
+super: "juu"
+then: "kisha"
+true: "kweli"
+undef: "batilisha"
+unless: "isipokuwa"
+until: "mpaka"
+when: "pale"
+while: "wakati"
+yield: "toa"
+
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE:  RUST  |
+# ----------------------
+# << Keywords >>
+
+_: ""
+abstract: "muhtasari"
+alignof: "alinio_la"
+as: "kama"
+become: "kuwa"
+box: "sanduku"
+break: "vunja"
+const: "daima"
+continue: "endelea"
+crate: "kizibo"
+do: "fanya"
+else: "vinginevyo"
+enum: "sura"
+extern: "nje"
+false: "uwongo"
+final: "mwisho"
+fn: "fn"
+for: "kwa"
+if: "kama"
+impl: "utekelezaji"
+in: "katika"
+let: "ruhusu"
+loop: "rudia"
+macro: "makro"
+match: "sawa"
+mod: "mod"
+move: "hamisha"
+mut: "badilisha"
```

Comment:
badilisha means change . mut stands for mutate(transform). try finding a good swahili word for that